Alcatraz/Controllers/ATZPluginWindowController.m

```diff
@@ -237,6 +251,11 @@ - (void)updatePredicate {
 
     [self.tableViewDelegate filterUsingPredicate:[NSCompoundPredicate andPredicateWithSubpredicates:predicates]];
     [self.tableView reloadData];
+
+	dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
+		// wait next run loop, otherwise there are no rows yet
+		[self loadImagesInVisibleRows];
+	});
```

Comment:
Because I don't load images when cell loaded, only after slight delay when scroll ends. So if I scroll quickly through list, I wont get all them in loading queue.
Actually this part of code loads images in first visible cells after changing filtering predicate, because scroll event wont be fired at this point.